src/core/core.adapters.js

```diff
@@ -30,20 +30,12 @@ function abstract() {
 /** @lends Chart._adapters._date */
 module.exports._date = {
 	/**
-	 * Returns a map of time formats for the supported units.
+	 * Returns a map of time formats for the supported formatting units.
+	 * This includes all scale tick units defined in Unit as well as 'datetime' used in the tooltip.
```

Comment:
Adapters should not know about `scale tick` and `tooltip` so can we change this comment for:

```
    /**
     * Returns a map of time formats for the supported formatting units defined
     * in Unit as well as 'datetime' representing a detailed date/time string.
     */
```

Reply:
done